public/app/plugins/datasource/influxdb/influx_query_model.test.ts

```diff
@@ -5,6 +5,33 @@ import InfluxQueryModel from './influx_query_model';
 describe('InfluxQuery', () => {
   const templateSrv = { replace: (val) => val } as TemplateSrv;
 
+  describe('interpolateQueryStr', () => {
+    // This is not the desired functionality!
```

Comment:
I expected this test to fail in 10.2.2 and return an un-escaped value, but it passed.